docs/trace_collection.md

```diff
@@ -92,6 +92,24 @@ class SampleApplication : Application() {
     ```kotlin
     span.setTag("http.url", url)
     ```
+7. Logging a throwable or an error message for a Span:
+
+    Using the Official Opentracing API
+
+    ```kotlin
+    span.log(mutableMapOf(Fields.ERROR_OBJECT to throwable))
+    ```
+    ```kotlin
+    span.log(mutableMapOf(Fields.MESSAGE to errorMessage))
+    ```
+    OR using our AndroidTracer helper methods
+
+    ```kotlin
+    AndroidTracer.logThrowable(forSpan, throwable)
+    ```
+    ```kotlin
+    AndroidTracer.logErrorMessage(forSpan, message)
```

Comment:
use the same variable name here (ie: `span`)  as above. using `forSpan` might confuse users.